src/log.h

```diff
@@ -33,14 +43,15 @@ class Log
         void    add( const QString& text );
 
         // Internal stuff
-        Log();
-        ~Log();
+        Log()  = default;
+        ~Log() = default;
```

Comment:
If you do this, you can also add **final** to the class def.

Reply:
Added